src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -540,7 +540,7 @@ public override void VisitMethod(IMethodSymbol symbol)
 
                             if (operatorKind == SyntaxKind.None)
                             {
-                                builder.Add(CreatePart(SymbolDisplayPartKind.MethodName, symbol, symbol.Name));
+                                builder.Add(CreatePart(SymbolDisplayPartKind.Operator, symbol, symbol.Name));
```

Comment:
>Operator [](http://example.com/codeflow?start=77&length=8)

It doesn't look like this change is backed by any test. And it looks like for conversions the same code path isn't changed.

Reply:
> It doesn't look like this change is backed by any test. And it looks like for conversions the same code path isn't changed.

It looks like there was no response for this comment

Reply:
reverted.